renovate.json

```diff
@@ -29,6 +29,11 @@
     {
       "groupName": "CI github-actions",
       "matchManagers": ["github-actions"]
+    },
+    {
+      "groupName": "Ignore package.json > engines",
```

Comment:
We don't have to target `engines`? 
https://github.com/renovatebot/renovate/discussions/13521#discussioncomment-1957196

Reply:
good find - a bit better to target it. Also, from their config
```json
      "description": "Don't bump engines field in package.json",
      "matchPackageNames": ["node"],
      "matchManagers": ["npm"],
      "matchDepTypes": ["engines"],
      "rangeStrategy": "auto"
```
`"rangeStrategy": "auto"` seems better than my `"enabled": false`